api/src/storage.rs

```diff
@@ -68,6 +68,7 @@ pub enum UpdateError {
 
 #[derive(Debug)]
 pub enum InsertError {
+    #[allow(dead_code)]
```

Comment:
that was the same error I was getting, and we said no allow dead code I think,  I think you should run rustup update locally, and the clippy error will appear